lib/cdo/rack/attack.rb

```diff
@@ -0,0 +1,59 @@
+require 'rack/attack'
+require 'active_support/core_ext/numeric/time'
+
+class Rack::Attack
+  # maximum requests per minute over a 4-minute stretch.
+  MAX_TABLE_READS_PER_MIN = 2 * 60
+  MAX_TABLE_WRITES_PER_MIN = 4 * 60
```

Comment:
These limits seem too low for legitimate use cases. I spoke with Alice and she is definitely willing to invest in much higher limits (especially at  launch) to make sure well-behaved apps don't break.

I expect that what we'll see is considerable burstiness in traffic-- a single app might have hundreds of reads and writes per second but few apps will be bursting at any given time.

Could we increase these limits based on some back of the envelope calculations about what "legitimate" apps might want to do?

If I imagine a full-class room chat or drawing app, we could easily expect 50 reads and writes per second in well behaved apps.